interfaces/builtin/media_hub.go

```diff
@@ -0,0 +1,301 @@
+	peer=(label=###PLUG_SECURITY_TAGS###),
+
+# Allow clients to pause all other sessions
+dbus (receive)
+    bus=session
+    path=/core/ubuntu/media/Service
+    interface=core.ubuntu.media.Service
+    member="PauseOtherSessions"
+	peer=(label=###PLUG_SECURITY_TAGS###),
+
+# Allow clients to query/modify service properties
+dbus (receive)
+    bus=session
+    path=/core/ubuntu/media/Service
+    interface=org.freedesktop.DBus.Properties
+    member="{Get,Set}"
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=session
+    path=/core/ubuntu/media/Service
+    interface=org.freedesktop.DBus.Properties
+    member=PropertiesChanged
+    peer=(label=###PLUG_SECURITY_TAGS###),
+`)
+
+var mediaHubConnectedSlotAppArmorClassic = []byte(`
+# Allow unconfined clients to interact with the player on classic
+dbus (receive)
+    bus=session
+    path=/core/ubuntu/media/Service
+    peer=(label=unconfined),
+
+dbus (receive)
+    bus=session
+    interface=org.freedesktop.DBus.Introspectable
+    peer=(label=unconfined),
+`)
+
+var mediaHubConnectedPlugAppArmor = []byte(`
+# Description: Allow using media-hub service. Reserved because this gives
+#  privileged access to the service.
+# Usage: reserved
+
+#include <abstractions/dbus-strict>
+
+# Allow clients to manage Player sessions
+dbus (send)
+    bus=session
+    path=/core/ubuntu/media/Service
+    interface=core.ubuntu.media.Service
+    member="{Create,Detach,Reattach,Destroy,CreateFixed,Resume}Session"
+	peer=(label=###SLOT_SECURITY_TAGS###),
+
+# Allow clients to pause all other sessions
+dbus (send)
+    bus=session
+    path=/core/ubuntu/media/Service
+    interface=core.ubuntu.media.Service
+    member="PauseOtherSessions"
+	peer=(label=###SLOT_SECURITY_TAGS###),
+
+# Allow clients to query service properties
+dbus (send)
+    bus=system
+    path=/core/ubuntu/media/Service
+    interface=org.freedesktop.DBus.Properties
+    member="{Get,Set}"
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (receive)
+   bus=system
+   path=/core/ubuntu/media/Service
+   interface=org.freedesktop.DBus.Properties
+   member=PropertiesChanged
+   peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (receive)
+    bus=system
+    path=/
+    interface=org.freedesktop.DBus.ObjectManager
+    peer=(label=unconfined),
+`)
+
+var mediaHubPermanentSlotSecComp = []byte(`
+getsockname
+recvmsg
+sendmsg
+sendto
+`)
+
+var mediaHubConnectedPlugSecComp = []byte(`
+getsockname
+recvmsg
+sendmsg
+sendto
+`)
+
+var mediaHubPermanentSlotDBus = []byte(`
+<policy user="root">
+    <allow own="core.ubuntu.media.Service"/>
+    <allow send_destination="core.ubuntu.media.Service"/>
+    <allow send_interface="core.ubuntu.media.Service"/>
+</policy>
+`)
+
+var mediaHubConnectedPlugDBus = []byte(`
+<policy context="default">
+    <deny own="core.ubuntu.media.Service"/>
+    <allow send_destination="core.ubuntu.media.Service"/>
+    <allow send_interface="core.ubuntu.media.Service"/>
+</policy>
```

Comment:
Update after talking with Simon: Basically since we're including media-hub as part of the unity8-session snap, nothing is blocking this access anyway for the permanentSlot/connectedPlug so there's no need to explicitly allow with these rules, so let's remove these two.